doc/nrf/app_dev/device_guides/nrf54l/cryptography.rst

```diff
@@ -0,0 +1,336 @@
+.. _ug_nrf54l_cryptography:
+
+nRF54L Series cryptography
```

Comment:
@mia-ko Isn't this applicable also to 54H?

As I believe it is in scope of 2.8 to have support for crypto PSA on both, are we planning to duplicate this page in both "working with" sections or will this be moved somewhere else (like in app dev)? @annwoj  FYI

Reply:
Crypto for 54H Series is not the same as for 54L Series.